dbos_transact/dbos.py

```diff
@@ -399,8 +417,13 @@ def execute_workflow_uuid(self, workflow_uuid: str) -> WorkflowHandle[Any]:
             raise DBOSWorkflowFunctionNotFoundError(
                 workflow_uuid, "Workflow function not found"
             )
-        with SetWorkflowUUID(workflow_uuid):
-            return self.start_workflow(wf_func, *inputs["args"], **inputs["kwargs"])
+        with DBOSContextEnsure():
+            ctx = assert_current_dbos_context()
+            request = status["request"]
+            assert request is not None
```

Comment:
Why do we want to assert this? Request can be `None` if the workflow is not from HTTP. For example, from Kafka and SQS.